utils/crypto_utils.h

```diff
@@ -20,16 +20,90 @@ extern "C" {
 #define MAXLINE 1024
 #define IMSI_LEN 15
 
-int get_device_id(const char *device_id);
-int get_aes_key(const uint8_t *key);
+/**
+ * @brief Gets device IMSI or physical ID
+ *
+ * @param[out] device_id Device id output
+ *
+ * @return #retcode_t
+ */
+retcode_t get_device_id(const char *device_id);
+
+/**
+ * @brief Gets aes key from hashchain
+ *
+ * @param[out] key Device key output
+ *
+ * @return #retcode_t
+ */
+retcode_t get_aes_key(const uint8_t *key);
+
+/**
+ * @brief Encrypt plaintext to ciphertext
+ *
+ * @param[in] plaintext The text to be encrypted
+ * @param[in] plaintext_len Plaintext length
+ * @param[out] ciphertext Ciphrtext
```

Comment:
Move output parameter in the end of the parameters.

Reply:
Already mark this at #54